### PR TITLE
G-API: Add parameters alpha and beta in tests on ConvertTo kernel

### PR DESCRIPTION
**Pull request by smirnov-alexey**
This pullrequest adds new parameters alpha and beta in tests on ConvertTo kernel

### CODE REVIEW

File: modules/gapi/test/cpu/gapi_core_tests_fluid.cpp [hunk 1]
```diff
@@ -142,6 +142,9 @@ INSTANTIATE_TEST_CASE_P(ConvertToFluid, ConvertToTest,
                                        cv::Size(1280, 720),
                                        cv::Size(640, 480),
                                        cv::Size(128, 128)),
+                                Values(3.0, 1.0, 0.5, 0.0, -1.0),
+                                Values(256.0, 32.0, 0.0, -128.0),
+                                Values(Tolerance_FloatRel_IntAbs(1e-5, 2).to_compare_f()),
```
Comment (smirnov-alexey) [inline modules/gapi/test/cpu/gapi_core_tests_fluid.cpp]: Fluid version is the only reason for tolerance. Seems like we have minor round issues with CV_32FC1 and alpha = 3, beta = {256, -128}

**Comment by smirnov-alexey** [comment]
@rgarnov , @dmatveev please, review

File: modules/gapi/test/cpu/gapi_core_tests_cpu.cpp [hunk 2]
```diff
@@ -369,11 +369,14 @@ INSTANTIATE_TEST_CASE_P(LUTTestCustomCPU, LUTTest,
...
 INSTANTIATE_TEST_CASE_P(ConvertToCPU, ConvertToTest,
-                        Combine(Values( CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1, CV_32FC1 ),
+                        Combine(Values(CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1, CV_32FC1),
                                 Values(CV_8U, CV_16U, CV_16S, CV_32F),
                                 Values(cv::Size(1280, 720),
                                        cv::Size(640, 480),
                                        cv::Size(128, 128)),
+                                Values(3.0, 1.0, 0.5, 0.0, -1.0),
+                                Values(256.0, 32.0, 0.0, -128.0),
+                                Values(Tolerance_FloatRel_IntAbs(1e-5, 2).to_compare_f()),
```
Comment (rgarnov) [inline modules/gapi/test/cpu/gapi_core_tests_cpu.cpp]: I believe there should be `AbsExact` comparison function (and for GPU as well)

Comment (smirnov-alexey) [reply modules/gapi/test/cpu/gapi_core_tests_cpu.cpp]: Done, thanks!

**Comment by smirnov-alexey** [comment]
@alalek ,please, merge.

File: modules/gapi/test/cpu/gapi_core_tests_cpu.cpp [hunk 3]
```diff
@@ -369,11 +369,14 @@ INSTANTIATE_TEST_CASE_P(LUTTestCustomCPU, LUTTest,
...
 INSTANTIATE_TEST_CASE_P(ConvertToCPU, ConvertToTest,
-                        Combine(Values( CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1, CV_32FC1 ),
+                        Combine(Values(CV_8UC1, CV_8UC3, CV_16UC1, CV_16SC1, CV_32FC1),
                                 Values(CV_8U, CV_16U, CV_16S, CV_32F),
                                 Values(cv::Size(1280, 720),
                                        cv::Size(640, 480),
                                        cv::Size(128, 128)),
+                                Values(3.0, 1.0, 0.5, 0.0, -1.0),
+                                Values(256.0, 32.0, 0.0, -128.0),
+                                Values(AbsExact().to_compare_f()),
```
Comment (alalek) [inline modules/gapi/test/cpu/gapi_core_tests_cpu.cpp]: > ConvertToGPU/ConvertToTest.AccuracyTest/1199
> ConvertToCPU/ConvertToTest.AccuracyTest/1199
> ConvertToFluid/ConvertToTest.AccuracyTest/959

Please don't explode testing system.
Reduce amount of test cases until coverage is on acceptable level.

Comment (smirnov-alexey) [reply modules/gapi/test/cpu/gapi_core_tests_cpu.cpp]: Done!

**Pull request merged**